modules/openapi-generator/src/main/resources/go-server/model.mustache

```diff
@@ -1,10 +1,19 @@
 {{>partial_header}}
 package {{packageName}}
+
 {{#models}}{{#imports}}
 {{#-first}}import (
 {{/-first}}	"{{import}}"{{#-last}}
 )
-{{/-last}}{{/imports}}{{#model}}{{#isEnum}}{{#description}}// {{{classname}}} : {{{description}}}{{/description}}
+{{/-last}}{{/imports}}
+
+import "errors" // FIXME: why not in #imports
```

Comment:
Not sure how to have the imports automatically added. Any help welcome.

Reply:
For this, you will have to look at `postProcessOperationsWithModels`

in modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/GoServerCodegen.java